src/OAuthClient/BibClient.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace NYPL\HoldRequestResultConsumer\OAuthClient;
+
+use NYPL\HoldRequestResultConsumer\Model\DataModel\Bib;
+use NYPL\Starter\APILogger;
+use NYPL\Starter\Config;
+
+class BibClient extends APIClient
+{
+    public static function getBibByIdAndSource($bibId = '', $nyplSource)
+    {
+        $url = Config::get('API_BASE_URL') . '/bibs/'. $nyplSource . '/' . $bibId;
```

Comment:
Perhaps it would be better to set the entire string as an environment parameter, like API_BIB_URL, instead of setting the entire string here?